cpp/kiss_icp/pipeline/KissICP.hpp

```diff
@@ -58,6 +58,7 @@ class KissICP {
     using Vector3dVectorTuple = std::tuple<Vector3dVector, Vector3dVector>;
 
 public:
+    explicit KissICP() : KissICP(KISSConfig()) {}
```

Comment:
This was intentionally deleted in https://github.com/PRBonn/kiss-icp/pull/294

Reply:
Okay, then I didn't fully understand what the problem in https://github.com/PRBonn/kiss-icp/pull/294 was. Can you explain again? Is this about the tbb max number of threads being set to default at compile time if a default constructor is provided?

Reply:
Basically yes, would be a matter of taste.  I kind of liked the idea of not allowing to construction the pipeline with default parameters.

Not a big deal though, but then we shall remove the hack off the TBB number of threads and make it a registration member, this uglyfy quite a bit the API while being more correct at the same time.